bot/utils/messages.py

```diff
@@ -33,10 +32,7 @@ async def get_discord_message(ctx: Context, text: str) -> Union[Message, str]:
     Conversion will succeed if given a discord Message ID or link.
     Returns `text` if the conversion fails.
     """
-    try:
-        text = await MessageConverter().convert(ctx, text)
-    except commands.BadArgument:
-        log.debug(f"Input '{text:.20}...' is not a valid Discord Message")
+    text = await MessageConverter().convert(ctx, text)
```

Comment:
I think you still need the try-catch here. Otherwise, this will raise an actual exception on every message without a parsable link. Just without logging was my original suggestion. Either return text in the except block or pass.